src/Traits/CollectionArrayAccess.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Softonic\GraphQL\Traits;
+
+trait CollectionArrayAccess
+{
+    public function offsetSet($offset, $value)
+    {
+        throw new \BadMethodCallException('Try using add() instead');
+    }
+
+    public function offsetExists($offset)
+    {
+        return isset($this->arguments[$offset]);
```

Comment:
`array_key_exists` checks if the key exists, while `isset` is going to return false when it exists but the value is null